rules/override/plugx.yara

```diff
@@ -0,0 +1,21 @@
+rule plugx : override {
+    meta:
+        malware_PlugX_config = "high"
```

Comment:
🎉

Reply:
this override could use a description; I can't figure out what the override is for or what these string values are.

Reply:
Yep, just a test. Removed in [`a4c70f4` (#487)](https://github.com/chainguard-dev/malcontent/pull/487/commits/a4c70f42836206fc80d72f275130f309a8a56037).

I'll work on a better rule in the next false-positive PR.